ingest-upsert-archive-folders/src/test/scala/uk/gov/nationalarchives/ingestupsertarchivefolders/testUtils/ExternalServicesTestUtils.scala

```diff
@@ -31,6 +32,8 @@ import scala.jdk.CollectionConverters.*
 
 class ExternalServicesTestUtils extends AnyFlatSpec with BeforeAndAfterEach with BeforeAndAfterAll with TableDrivenPropertyChecks with MockitoSugar {
 
+  case class TestIdentifier(name: String, value: String)
```

Comment:
How come we added this?

Reply:
The `Table` we're using for the tests has a mix of Preservica `Identifier` and Dynamo `Identifier`. This was fine when they were the same thing but now it's got a bit messy. I couldn't get it to compile in any kind of neat way so I introduced this as a sort of "bridge" between the two for testing purposes.